### PR TITLE
Fix boot from eMMC.

### PR DESCRIPTION
**Pull request by camachat**

### CODE REVIEW

File: board/BeagleBone/overlay/root/copy-to-emmc.sh [hunk 1]
```diff
@@ -29,9 +29,9 @@ dd if=/dev/zero of=/dev/mmcsd1 bs=64k count=1
 echo
 echo 'Creating MSDOS FAT12 boot partition on eMMC'
```
Comment (rozmansi) [inline board/BeagleBone/overlay/root/copy-to-emmc.sh]: FAT12 => FAT16 ?

Comment (kientzle) [inline board/BeagleBone/overlay/root/copy-to-emmc.sh]: @camachat:  Could you please fix this comment?  Then we can merge it.  Thanks!

Comment (camachat) [inline board/BeagleBone/overlay/root/copy-to-emmc.sh]: Surely, I will fix it

**Comment by kientzle** [comment]
Thanks!

**Pull request merged**